agreement/asyncVoteVerifier_test.go

```diff
@@ -38,7 +46,7 @@ func (fp *expiredExecPool) BufferSize() (length, capacity int) {
 	return
 }
 
-// Test async vote verifier against a full execution pool.
+// Test async vote verifier against a canceled execution pool.
```

Comment:
why is it cancelled? `MakeAsyncVoteVerifier` creates a new verifier without cancelling it

Reply:
Because it creates an `expiredExecPool`, which returns `context.Canceled` for `EnqueueBacklog`.
The full pool blocks the call `EnqueueBacklog`, but here it behaves like canceled, does not block, and returns `context.Canceled`.